resources/EPvalidator.rb

```diff
@@ -584,51 +589,51 @@ def self.run_validator(hpxml_doc)
         "ArrayAzimuth" => one,
         "ArrayTilt" => one,
         "MaxPowerOutput" => one,
-        "InverterEfficiency" => one, # PVWatts default is 0.96
-        "SystemLossesFraction" => one, # PVWatts default is 0.14
+        "InverterEfficiency" => zero_or_one,
+        "SystemLossesFraction" => zero_or_one,
       },
 
       # [ClothesWasher]
       "/HPXML/Building/BuildingDetails/Appliances/ClothesWasher" => {
         "SystemIdentifier" => one, # Required by HPXML schema
         "[Location='living space' or Location='basement - conditioned' or Location='basement - unconditioned' or Location='garage']" => one,
-        "ModifiedEnergyFactor | IntegratedModifiedEnergyFactor" => one,
-        "RatedAnnualkWh" => one,
-        "LabelElectricRate" => one,
-        "LabelGasRate" => one,
-        "LabelAnnualGasCost" => one,
-        "Capacity" => one,
+        "ModifiedEnergyFactor | IntegratedModifiedEnergyFactor" => zero_or_one,
+        "RatedAnnualkWh" => zero_or_one,
+        "LabelElectricRate" => zero_or_one,
+        "LabelGasRate" => zero_or_one,
+        "LabelAnnualGasCost" => zero_or_one,
+        "Capacity" => zero_or_one,
```

Comment:
Instead, could do:

```
      "/HPXML/Building/BuildingDetails/Appliances/ClothesWasher" => {
          ...  
          "RatedAnnualkWh" => zero_or_one, # See [ClothesWasher=Detailed]
      }

     # [ClothesWasher=Detailed]
     "/HPXML/Building/BuildingDetails/Appliances/ClothesWasher[RatedAnnualkWh]" => {
                   "LabelGasRate" => _one,
                   "Capacity" => one,
                   ...
     }
```

And then only have to check if rated_annual_kwh is nil in measure.rb.